atrium-repo/README.md

```diff
@@ -0,0 +1,10 @@
+# ATrium Repo
+
+[![](https://img.shields.io/crates/v/atrium-repo)](https://crates.io/crates/atrium-repo)
+[![](https://img.shields.io/docsrs/atrium-repo)](https://docs.rs/atrium-repo)
+[![](https://img.shields.io/crates/l/atrium-repo)](https://github.com/sugyan/atrium/blob/main/LICENSE)
+[![Rust](https://github.com/sugyan/atrium/actions/workflows/api.yml/badge.svg?branch=main)](https://github.com/sugyan/atrium/actions/workflows/api.yml)
```

Comment:
Since this package also contains tests, you should also add `workflows/repo.yml` and this should be a link to it.